pysimplesoap/client.py

```diff
@@ -37,6 +37,7 @@
                      get_local_name, get_namespace_prefix, TYPE_MAP, urlsplit, \
                      iteritems
 from .wsse import UsernameToken
+from urllib3.util import make_headers
```

Comment:
Yes. This will work with all transports as it will add login credentials to the http headers if username and password are provided